### PR TITLE
magento/devdocs#7366: GraphQL: Document password change mutations. Mutation: resetPassword.

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds a new topic about `resetPassword` mutation in scope of https://github.com/magento/devdocs/issues/7364.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/graphql/mutations/reset-password.html

## Links to Magento source code

- https://github.com/magento/magento2/pull/27876

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.4/graphql/mutations/reset-password.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+The `resetPassword` mutation must contain the following arguments:
+
+Argument | Type | Description
+--- | --- | ---
+`email` | String! | Specifies the customer account that needs a password reset
+`resetPasswordToken` | String! | A runtime token
+`newPassword` | String! | The new password
+
+{:.bs-callout-info}
+Your new password must be six or more characters in length without spaces.
```
Comment (keharper) [inline src/guides/v2.4/graphql/mutations/reset-password.md:51]: Does the value of Customers > Customer Configuration > Password Options > Minimum Password Length determine this? Would it be better to say that the new password must meet the password policies set for the store?

File: src/guides/v2.4/graphql/mutations/reset-password.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+## Input arguments
+
+The `resetPassword` mutation must contain the following arguments:
+
+Argument | Type | Description
+--- | --- | ---
+`email` | String! | Specifies the customer account that needs a password reset
+`resetPasswordToken` | String! | A runtime token
```
[the remark this answers is not in the document]
Comment (atwixfirster) [reply src/guides/v2.4/graphql/mutations/reset-password.md:47]: from `email`.... or from `customer_entity`.`rp_token`

[the remark this answers is not in the document]
Comment (atwixfirster) [reply src/guides/v2.4/graphql/mutations/reset-password.md]: **fixed**

Could you please review?

File: src/guides/v2.4/graphql/mutations/reset-password.md (hunk 1 above)
Comment (atwixfirster) [reply src/guides/v2.4/graphql/mutations/reset-password.md]: Good point! 

**fixed**

**Comment by keharper** [comment]
Since this PR is still waiting for a technical review, I commented out the links to `resetPassword` in #7424 so that the PR can be merged. 

In this PR, update src/guides/v2.4/graphql/mutations/request-password-reset-email.md by uncommenting those links.

**Comment by keharper** [comment]
@atwixfirster I need you to fix the links to this topic as described previously.

**Comment by atwixfirster** [comment]
> @atwixfirster I need you to fix the links to this topic as described previously.

**done**

**Comment by keharper** [comment]
running tests

**Pull request merged**